catalogs/mississippi-surface-water/build_catalog.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+import argparse
+from datetime import datetime, time, date
+from urllib.parse import urlparse
+
+import boto3
+from pystac import (
+    Asset,
+    CatalogType,
+    Collection,
+    Extent,
+    Item,
+    SpatialExtent,
+    TemporalExtent,
+)
+from shapely.geometry import Polygon, mapping
+
+
+if __name__ == "__main__":
+    """Constructs STAC Catalog for surface water observed on a monthly basis around the Mississippi river
+
+    Will include all images with the extension `tif`
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--jrc-monthly-root-s3",
+        required=True,
+        help="Bucket+key path to a directory containing GLOFIMR flood ids",
```

Comment:
I believe this docstring is also still incorrect.

Reply:
Oof. Fixed now